src/Tool/Tool.csproj

```diff
@@ -24,7 +24,22 @@
     <ContentTargetFolders>\</ContentTargetFolders>
     <ApplicationIcon />
   </PropertyGroup>
-  
+
+  <ItemGroup>
```

Comment:
Can you validate that this works and is correctly included in conda packages? Want to make sure we're not accidentally introducing new dependencies from outside of conda environments.

+@IsraelMiles for an example of testing conda dependencies.

Reply:
This needs to be manually tested in a Mac environment.

Reply:
Decided offline that the pipeline validation along with the validation provided by Israel is sufficient for now.